### ISSUE TITLE
Parameters with Instance normalization and layer normalization

### ISSUE THREAD

**Issue by fengwang** [JaeDukSeo/Daily-Neural-Network-Practice-2#1]
Hello, I went through your post 'Implementing Spatial Batch / Instance / Layer Normalization in Tensorflow'. In that post, it has been pointed out that 

> Batch Norm → Take mean and variance respect to channel (1,1,1,c)
> Layer Norm → Take mean and variance respect to batch (b,1,1,1)
> Instance Norm → Take mean and variance respect to batch/channel (b,1,1,c)

But with keras/TF-2.4.1, the code below

```python
from tensorflow.keras.layers import Input
from tensorflow.keras.models import Model
from tensorflow.keras.layers import BatchNormalization, LayerNormalization
from instance_normalization import InstanceNormalization


print( 'With BN' )
inp = Input( shape=( 3, 5, 7, 11 ) )
ll = BatchNormalization()( inp )
m = Model( inp, ll )
m.summary()


print( 'With IN' )
inp = Input( shape=( 3, 5, 7, 11 ) )
ll = InstanceNormalization()( inp )
m = Model( inp, ll )
m.summary()


print( 'With LN' )
inp = Input( shape=( 3, 5, 7, 11 ) )
ll = LayerNormalization()( inp )
m = Model( inp, ll )
m.summary()
```

gives out result

```
Model: "model"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
input_1 (InputLayer)         [(None, 3, 5, 7, 11)]     0         
_________________________________________________________________
batch_normalization (BatchNo (None, 3, 5, 7, 11)       44        
=================================================================
Total params: 44
Trainable params: 22
Non-trainable params: 22
_________________________________________________________________
With IN
Model: "model_1"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
input_2 (InputLayer)         [(None, 3, 5, 7, 11)]     0         
_________________________________________________________________
instance_normalization (Inst (None, 3, 5, 7, 11)       2         
=================================================================
Total params: 2
Trainable params: 2
Non-trainable params: 0
_________________________________________________________________
With LN
Model: "model_2"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
input_3 (InputLayer)         [(None, 3, 5, 7, 11)]     0         
_________________________________________________________________
layer_normalization (LayerNo (None, 3, 5, 7, 11)       22        
=================================================================
Total params: 22
Trainable params: 22
Non-trainable params: 0
```

Could you please quickly review this difference?

I mean, it seems in keras, the layer normalization layer takes variance and mean from the channel (1, 1, 1, c), and instance normalization takes variance and mean from the whole instance (1,1, 1, 1).

Updating the input shape from (3, 5, 7, 11) to (5, 7, 11) gives same result.
Instance normalization is taken from [tf.contrib](https://github.com/keras-team/keras-contrib/blob/master/keras_contrib/layers/normalization/instancenormalization.py)
